src/app/community/_components/Comment.tsx

```diff
@@ -100,7 +106,7 @@ export default function Comment({
 
   const timeAgo = calculateTimeDifference(createdTimeToDate);
   return (
-    <div className={cn('container')}>
+    <div className={cn('container')} ref={ref}>
       <ProfileImage profileImage={profile || null} />
```

Comment:
```suggestion
      <ProfileImage profileImage={profile && profile} />
```
사실 여기도 컴포넌트로 null을 넣어주는게 아닌 옵셔널로 받아도 괜찮을거같다는 개인적인 생각이 듭니다